test/validation/org/apache/cassandra/bridges/ccmbridge/CaptureNodeLogs.java

```diff
@@ -0,0 +1,126 @@
+/*
+ *      Copyright (C) 2012-2014 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package org.apache.cassandra.bridges.ccmbridge;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+import org.apache.commons.io.FileUtils;
+
+public class CaptureNodeLogs
+{
+    public void preserveClusterLogs(File ccmDir, File CASSANDRA_DIR, String DEFAULT_CLUSTER_NAME, int nodeCount, String testName){
```

Comment:
Move this back into CCMBridge. I said only the last three methods, not four, needed extracted.